api.planx.uk/package.json

```diff
@@ -3,9 +3,10 @@
   "license": "MPL-2.0",
   "private": true,
   "packageManager": "pnpm@8.6.6",
+  "type": "module",
   "dependencies": {
     "@airbrake/node": "^2.1.8",
-    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#6b2fd26",
+    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#dd24b0a",
```

Comment:
This may now be an outdated commit hash - worth checking!

Reply:
Bumped this to latest and squashed into same commit as before :)